public/data/imran.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "Imran Khan",
+    "type": "personal",
+    "bio": "Open Source Enthusiast !",
+    "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/imran201khan",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/imrankhan",
+        "icon": "Linkden"
```

As it is your Twitter profile so change the icon from LinkedIn to Twitter.

```suggestion
        "icon": "twitter"
```